### DISCUSSION TITLE
Puffer, docker and templates

### DISCUSSION THREAD

**Discussion by loranger** [pufferpanel/pufferpanel#1128] [Q&A] [answered]
Hello !

Does anyone know if I can mount a volume from my local drive to the dockerized pufferpanel in order to have, edit and store templates ?

I tried

```yaml
    volumes:
      ...
      - ./templates:/var/lib/pufferd/templates
```

But it does not seems to work anymore…

**Comment by nepcore** [comment, marked as answer]
Templates aren't stored as files anymore since 2.0, they are stored in the database and managed using the panel

**Reply by loranger** [reply]
All right, thanks !
